myuw/templates/handlebars/academic_calendar.html

```diff
@@ -1,60 +0,0 @@
-	{% load templatetag_handlebars %}
-    {% tplhandlebars "calendar_events" %}
-    <div class="container academic_cal-container">
-
-
-        <div class="row">
-            <div class="col-md-8">
-
-                <div class="acal-main">
-                     <h2 id="main" class="heading-page">Academic Calendar</h2>
```

Comment:
@fanglinfang in this file, I changed quite a bit of HTML starting here on **line 10 - line 26** and another change on **line 39**.